### PR TITLE
DEPLOY-456 / DEPLOY-528 : Add nginx-ingress as a requirement to Infra 1.0.0

### PR DESCRIPTION
**Pull request by sergiuv2020**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +81,138 @@ kubectl delete namespace $DESIREDNAMESPACE
...
+</p>
+</details>
+
+<details>
+<summary>Option 2</summary>
+<p>
+
+If you
+
+* created the cluster in AWS using [kops](https://github.com/kubernetes/kops/)
```
Comment (rgauss) [inline README.md]: If you drop the reference to kops here and just say in AWS it will apply to EKS as well.

File: README.md [hunk 2]
```diff
@@ -82,6 +81,138 @@ kubectl delete namespace $DESIREDNAMESPACE
...
+* have a matching SSL/TLS certificate stored in [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/)
+* are using a zone in [Amazon Route 53](https://aws.amazon.com/route53/)
+
+Kubernetes' [External DNS](https://github.com/kubernetes-incubator/external-dns)
```
Comment (rgauss) [inline README.md]: We should point out that this is currently alpha functionality.

**Review by rgauss — CHANGES_REQUESTED** [review]
Minor docs changes.

**Comment by sergiuv2020** [comment]
Sounds good +1

**Review by rgauss — APPROVED** [review]

**Comment by nicdoye** [comment]
image

**Review by nicdoye — APPROVED** [review]
image

**Pull request merged**